src/app/elevation/qgselevationprofilewidget.cpp

```diff
@@ -411,6 +414,28 @@ QgsElevationProfileWidget::QgsElevationProfileWidget( const QString &name )
 
   mOptionsMenu->addSeparator();
 
+  // show Subsections Indicator Action
+  // create a default simple symbology
+  mSubsectionsSymbol.reset( QgsLineSymbol::createSimple(
+    { { QStringLiteral( "line_color" ), QStringLiteral( "#ff0000" ) },
+      { QStringLiteral( "line_width" ), QStringLiteral( "0.5" ) },
+      { QStringLiteral( "capstyle" ), QStringLiteral( "flat" ) }
+    }
+  ) );
```

Comment:
Shouldn't this use the method from QgsPlot instead?

Reply:
Done.